aviator.yml

```
@@ -0,0 +1,17 @@
+spruce:
+  - base: ci/infra/meta.yml
+    prune:
+      - meta
+    merge:
+      - with_in: ci/infra/
+        regexp: ".*yml"
+        except:
+          - credentials.yml.example
+          - credentials.yml
+      - with_in: ci/infra/jobs/build_amis/
```

```suggestion
      - with_in: ci/infra/jobs/build_amis/
         regexp: ".*yml"
```
just in case any one adds anything else in there